usr/sbin/pkcsslotd/socket_server.c

```diff
@@ -321,16 +353,21 @@ static int client_socket_notify(int events, void *private)
 
     if (socket < 0)
         return -ENOTCONN;
-
+#if defined(_AIX)
+    if (events & (POLLHUP | POLLERR)) {
```

Comment:
instead of ifdefing all those event cases, you could use defines like the following:

```
#if defined(_AIX)
#define EPOLLHUP  POLLHUP 
...
#endif
```